README.md

```diff
@@ -9,11 +9,11 @@ Ember Sinon QUnit
 This addon integrates [Sinon](http://jhawk.co/sinonjs) & [Ember-QUnit](http://jhawk.co/ember-qunit) 
 via [Ember-Sinon](http://jhawk.co/ember-sinon), as inspired by [Sinon-QUnit](http://jhawk.co/sinon-qunit).
 
-Why not simply use Ember-Sinon? Two reasons:
+Why not simply use Ember-Sinon alone? Two reasons:
```

Comment:
This is very nitpicky, but I generally prefer to refer to `ember-*` packages via their true name, case sensitive and all. So, in this line, Ember-Sinon would become `ember-sinon`, as that's the correct package name. I feel it helps with clarity. What do you think?

Reply:
I go back and forth on this, but I'm not strongly opposed to it.

Reply:
Done.